docs/variants/novacustom_nv4x/building.md

```diff
@@ -15,40 +15,52 @@ This documents describes the procedure for compiling coreboot for NovaCustom NV4
 
 The easiest way to build coreboot is to use the official Docker image.
 
-Obtain the image:
+1. Obtain the image:
 
-```bash
-docker pull coreboot/coreboot-sdk:0ad5fbd48d
-```
+    ```bash
+    docker pull coreboot/coreboot-sdk:0ad5fbd48d
+    ```
 
-Clone the coreboot repository:
+1. Clone the coreboot repository:
 
-```bash
-git clone https://review.coreboot.org/coreboot.git
-```
+    ```bash
+    git clone https://review.coreboot.org/coreboot.git
+    ```
 
-Navigate to the source code directory and checkout to the desired revision:
+    Navigate to the source code directory and checkout to the desired revision:
 
-> Replace the REVISION with one of the:
-> - `novacustom_nv4x/release` for the latest released version
-> - `novacustom_nv4x_vVERSION` (e.g. `v1.2.1`) for the given release
+    > Replace the REVISION with:
+    >
+    > - `clevo/release` for the latest released version
+    > - `novacustom_nv4x_vVERSION` (e.g. `v1.3.0`) for the given release
 
-```bash
-cd coreboot
-git remote add dasharo https://github.com/dasharo/coreboot.git
-git submodule update --init --recursive --checkout
-git fetch dasharo
-git checkout REVISION
-```
+    ```bash
+    cd coreboot
+    git remote add dasharo https://github.com/dasharo/coreboot.git
+    git submodule update --init --recursive --checkout
+    git fetch dasharo
+    git checkout REVISION
+    ```
 
-Build the firmware:
+1. Start the coreboot-sdk Docker container:
 
-```bash
-./build.sh build
-```
+    ```bash
+    docker run --rm -it -u $UID \
+       -v $PWD:/home/coreboot/coreboot \
+       -w /home/coreboot/coreboot \
+       coreboot/coreboot-sdk:0ad5fbd48d /bin/bash
+    ```
+
+1. Build the firmware:
+
+    ```bash
+    cp configs/config.novacustom_ns5x .config
+    make olddefconfig
+    make
+    ```
 
 The resulting coreboot image will be placed in
```

Comment:
I've tested the procedure and after `make` finishes there is no `coreboot.rom` in `build\`.

[Link to my task](https://jira-3mdeb.atlassian.net/browse/NCM-529)

Reply:
Pushed a fix, please retest. Now it built successfully for me

Reply:
@PrzemyslawBanasiak13 @sulewskiprzemyslaw Have we retested that?
We need all of the documentation changes live before publishing NV4X release

Reply:
@macpijan @mkopec I've just tested the whole procedure and it works fine